base_business_document_import/tests/test_business_document_import.py

```diff
@@ -9,13 +9,7 @@
 class TestBaseBusinessDocumentImport(TransactionCase):
     def test_match_partner(self):
         partner1 = self.env["res.partner"].create(
-            {
-                "name": "Total SA",
-                "supplier": False,
-                "customer": False,
-                "ref": "TOTAL",
-                "website": "www.total.com",
-            }
+            {"name": "Total SA", "ref": "TOTAL", "website": "www.total.com"}
```

Comment:
Prefer a fake company :wink: :fuelpump:

Reply:
You're right, let's be less opinionated :)